### PR TITLE
Added basic exception handling

### PR DESCRIPTION
**Pull request by arcticbarra**
This solves #2. If there are any other exceptions you think there should be added, I'll be happy to add them.

Cheers

### CODE REVIEW

File: hfc.py [hunk 1]
```diff
@@ -1,4 +1,8 @@
-import requests
+try:
+    import requests
+except Exception:
```
Comment (kerk12) [inline hfc.py]: Please change it into ImportError. Exception catches every exception occurring (including library errors, even if a library is installed). Try adding it to an except statement right before `except Exception`, for example:
```
try:
    import requests
except ImportError:
    print("You're missing requests module. Please install it using pip")
    exit(1)
except Exception:
    print("An exception has occured while importing the requests module.")
    exit(1)
```

File: hfc.py [hunk 2]
```diff
@@ -17,12 +21,16 @@ def __str__(self):
...
+
 Licenced under GNU/GPLv3
-    
-""")
 
-    gh_name = input("Please enter your github username: ")
+""")
+    gh_name = ""
+    while True:
+        if gh_name == "":
```
Comment (kerk12) [inline hfc.py]: I think it's better this way:
```
while gh_name == "":
    gh_name = input("Please enter your github username: ")
```

**Comment by arcticbarra** [comment]
Done @kerk12

**Comment by arcticbarra** [comment]
Awesome! 👍

**Pull request merged**